cpptests/t_client.cpp

```diff
@@ -1,20 +1,22 @@
 #include <gtest/gtest.h>
 #include <cstdlib>
 #include <string>
+#include <cassert>
+#include <climits>
+#include <netdb.h>
+
 #include "hiredis.h"
 #include "common.h"
 
-
-using namespace hiredis;
+namespace hiredis {
 
 class Client {
 public:
-    operator redisContext*() {
+    redisContext* operator*() {
```

Comment:
Why?

Reply:
I ended up adding it later somewhere else. So now we have both!